API_Doc/creditrisk_api_doc.py

```diff
@@ -0,0 +1,335 @@
+"""Generated API Documentation sample using doc_writer_sample.py."""
+    
+doc = {
```

Comment:
If you try to do: 
```
from pyld import jsonld
print(jsonld.expand(doc))
```
At some point close to the end of the print:
```
 {'http://www.w3.org/ns/hydra/core#description': [{'@value': 'The Borrower Class'}],
      'http://www.w3.org/ns/hydra/core#title': [{'@value': 'borrower'}],
      'http://www.w3.org/ns/hydra/core#property': [{'@id': 'http://localhost:8080/creditrisk_api/vocab?resource=EntryPoint/Borrower',  <<<<<
```
this link is wrong. I don't know if it is a problem of the ApiDoc or of the doc_maker